src/Nethermind/Nethermind.Cli/NodeManager.cs

```diff
@@ -119,14 +115,13 @@ public async Task<JsValue> PostJint(string method, params object[] parameters)
             return returnValue;
         }
 
-        public async Task<string> Post(string method, params object[] parameters)
+        public async Task<string?> Post(string method, params object[] parameters)
         {
             return await Post<string>(method, parameters);
         }
 
-        public async Task<T> Post<T>(string method, params object[] parameters)
+        public async Task<T?> Post<T>(string method, params object[] parameters)
         {
-            T result = default;
```

Comment:
Unfortunately, you cannot *assign* default to `T` because `T` is non-nullable.  By having separate returns for the different code paths (see below) you can avoid this compiler confusion and get it to stop complaining.  I know some teams have a policy of "all functions have a single return" which I'm not a fan of personally, but would prevent this fix and instead require something else.

Reply:
I became a fan of it few months ago, helps a lot to read code later